picocolors.js

```diff
@@ -1,12 +1,15 @@
-let argv = process.argv || [],
-	env = process.env
+let p = process || {},
+	argv = p.argv || [],
+	env = p.env || {},
+	out = p.stdout || {}
+let ne = v => v != null && String(v).length > 0
 let isColorSupported =
-	!("NO_COLOR" in env || argv.includes("--no-color")) &&
-	("FORCE_COLOR" in env ||
+	!(ne(env.NO_COLOR) || argv.includes("--no-color")) &&
+	(ne(env.FORCE_COLOR) ||
 		argv.includes("--color") ||
-		process.platform === "win32" ||
-		(require != null && require("tty").isatty(1) && env.TERM !== "dumb") ||
-		"CI" in env)
+		p.platform === "win32" ||
+		(out.isTTY && env.TERM !== "dumb") ||
```

Comment:
`out` is used only once.
Could be shorter to write
`(p.stdout || {}).isTTY && ...`

Reply:
Good point, missed that